addons/account/models/account_move.py

```diff
@@ -3866,6 +3866,11 @@ def to_tuple(t):
         # Add the domain and order by in order to compute the cumulated balance in _compute_cumulated_balance
         return super(AccountMoveLine, self.with_context(domain_cumulated_balance=to_tuple(domain or []), order_cumulated_balance=order)).search_read(domain, fields, offset, limit, order)
 
+    def export_data(self, fields_to_export):
+        order = self._order + ', id'
+        domain = ('&', ('account_id', 'in', tuple(self.mapped('account_id').ids)), '&', ('display_type', 'not in', ('line_section', 'line_note')), ('parent_state', '=', 'posted'))
```

Comment:
Where does this domain come from?

Reply:
We need the account ids from the selected lines, that can be from different accounts, in order to compute de cumulated balance

Reply:
It is basically the same domain that come from output of `search_read` above, but as we can select lines from several accounts, I just added account.ids in the domain.

Reply:
And why not `[('id', 'in', self.ids)]`?

Reply:
Because in order to calculate the cumulated balance, we need the balance of all lines linked to the specified account, not just the balance of selected lines.

<img width="1177" alt="Screenshot 2022-11-07 at 16 28 46" src="https://user-images.githubusercontent.com/85162663/200349730-cc5a61b6-7794-4cac-a475-b4212c19dfb1.png">

Reply:
You need the balance of the lines in the original domain.
This feature cannot work with the current API, and we are removing the cumulated balance in future versions.

Everything needed to cumulated balance should be in accounting reports.

Reply:
> You need the balance of the lines in the original domain.

I don't understand why I would need the balance of the lines. Here is the domain that comes as output from search_read when unfold a line `['&', ['account_id', '=', 6], '&', ['display_type', 'not in', ['line_section', 'line_note']], ['parent_state', '=', 'posted']]`, why the output domain from export_data should be different? Plus, I just try that and the result is not the one expected as you can see in the image attached above.

> This feature cannot work with the current API, and we are removing the cumulated balance in future versions.

You mean that we finally don't fix ?